Gruntfile.js

```diff
@@ -83,9 +86,30 @@ module.exports = function( grunt ) {
 		}
 	} );
 
+	// Generating a JSON file that holds the icons SVG data, with each icon name as a key.
+	grunt.registerTask( 'json', () => {
+		const rootPath = path.resolve( __dirname, './' ),
+			jsonPath = path.join( rootPath, 'eicons.json' ),
+			configPath = path.join( rootPath, 'config.json' ),
```

Comment:
They are both `json` files, so please find better names for the variables

Reply:
Done.